frame/lottery/src/lib.rs

```diff
@@ -442,12 +453,12 @@ impl<T: Config> Pallet<T> {
 						Error::<T>::AlreadyParticipating
 					);
 				}
+				participating_calls.try_push(call_index).map_err(|_| Error::<T>::TooManyCalls)?;
```

Comment:
I think we need to rerun the benchmarks, right? @shawntabrizi

Reply:
The benchmark bot can probably not push to my repo, will put this in the next MR since it is currently not live.